ostd/src/mm/page/mod.rs

```diff
@@ -123,11 +123,18 @@ impl<M: PageMeta> Page<M> {
     ///
     /// # Safety
     ///
-    /// The physical address must represent a valid page and the caller must already hold one
-    /// reference count.
+    /// The physical address must represent a valid page and the caller must ensure the corresponding
+    /// virtual address points to an initialized metadata slot by holding a reference count priorly.
     pub(in crate::mm) unsafe fn inc_ref_count(paddr: Paddr) {
```

Comment:
The function bodies of the two `inc_ref_count` functions are non-trivial and exactly the same. Need to avoid this duplicated code.